services/explorer/src/utils/TransactionRow.tsx

```diff
@@ -181,3 +188,49 @@ export const toNodeTableRow = ({
     blocksProduced: pbftCount.toLocaleString('en-US'),
   };
 };
+
+export const toHolderTableRow = ({
```

Comment:
If it's holders specific let's move it to a specific file.

Reply:
Like on the types above, all the other transformation methods are listed here too. Should I split them near each table? I think it is easier to find them like this near its rendering component

Reply:
Yeah. Definitely should be split but RN I'm referring just to the Holders part because this is a `TransactionRow` file